package.json

```diff
@@ -15,10 +15,12 @@
     "@expo/vector-icons": "^13.0.0",
     "@react-navigation/bottom-tabs": "^6.4.0",
     "@react-navigation/native": "^6.0.13",
+    "@tanstack/react-query": "^4.10.1",
     "expo": "~46.0.9",
     "expo-device": "^4.3.0",
     "expo-notifications": "~0.16.1",
     "expo-status-bar": "~1.4.0",
+    "moment": "^2.29.4",
```

Comment:
![image](https://user-images.githubusercontent.com/42850232/200965913-949ff43c-fe33-4d1f-b5cf-be3aa0374e9c.png)

Reply:
😬

Reply:
On a more serious note, moment is deprecated.
I suggest you use one of these (in my order of preference):
1. https://day.js.org/
2. https://moment.github.io/luxon/#/?id=luxon